core/modules/entity/entity.class.inc

```diff
@@ -177,13 +177,54 @@ interface EntityInterface {
    * @since 1.13.0 Method added.
    */
   public function getFieldValues($field_name, $value_key = 'value', $langcode = NULL);
+
+  /**
+   * Returns the revision identifier of the entity.
+   *
+   * @return
+   *   The revision identifier of the entity, or NULL if the entity does not
+   *   have a revision identifier.
+   */
+  public function getRevisionId();
+
+  /**
+   * Checks if this entity is the active revision.
+   *
+   * @return bool
+   *   TRUE if the entity is the active revision, FALSE otherwise.
+   */
+  public function isActiveRevision();
+
+  /**
+   * Gets active revision for this entity.
+   */
+  public function getActiveRevision();
+
+  /**
+   * Sets this revision as the active revision.
+   */
+  public function setActiveRevision();
```

Comment:
Needs `@param` docblock.

Reply:
I dont get this. THere are no parameters to this function?

Reply:
Sorry, my mistake. I thought there was a boolean TRUE/FALSE parameter that was missing.